src/qt/bitcoingui.cpp

```diff
@@ -512,16 +512,18 @@ void BitcoinGUI::createMenuBar()
 
     QMenu* window_menu = appMenuBar->addMenu(tr("&Window"));
```

Comment:
15013 is empty

Reply:
This one changed sleep(2) to sleep(3) but in dash it has since been changed to 4
Looks like it was fixed here 213216c924 ("test: p2p_timeouts - Wait a second longer before timeout validations (#3619)", 2020-07-20)

Reply:
In that case, that commit should be dropped.